### ISSUE TITLE
airflow support for pipewrench

### ISSUE THREAD

**Issue by SandishKumarHN** [Cargill/pipewrench#28]
airflow support for pipewrench

**Comment by afoerster** [comment]
@SandishKumarHN this will be a brand new set of templates?

**Comment by SandishKumarHN** [comment]
Yes, in airflow task-id would be like this example-> sqoop_create_exec_{{ conf.source_database.name }}_{{ table.source.name }} and filenames would be same but .py or we can do like examples-> create_execute_sqoop_parquet_hdfs_impala.py and clean_sqoop_parquet_hdfs_impala.py

**Comment by afoerster** [comment]
I'm not sure I'm following, share a WIP branch when you get a rough outline.
